clang/lib/Driver/Driver.cpp

```diff
@@ -610,7 +610,8 @@ static llvm::Triple computeTargetTriple(const Driver &D,
     if (A->getOption().matches(options::OPT_m64) ||
         A->getOption().matches(options::OPT_maix64)) {
       AT = Target.get64BitArchVariant().getArch();
-      if (Target.getEnvironment() == llvm::Triple::GNUX32)
+      if (Target.getEnvironment() == llvm::Triple::GNUX32 ||
```

Comment:
Optional: you could simplify the pattern  with `llvm::is_contained({...}, X)`

Reply:
To be honest, I don't have the energy to rewrite it all manually, especially that these files don't seem to be clang-formatted consistently.